deploy/cloudbuild-release.yaml

```diff
@@ -2,27 +2,14 @@
 # see: https://cloud.google.com/container-builder/docs/configuring-builds/substitute-variable-values#using_default_substitutions
 steps:
 
-# Download the latest version of the builder image
-  - name: 'gcr.io/cloud-builders/docker'
-    entrypoint: 'bash'
-    args:
-    - '-c'
-    - |
-      docker pull gcr.io/$PROJECT_ID/skaffold-builder:latest || exit 0
-# until https://github.com/GoogleCloudPlatform/cloud-builders/issues/253 is fixed
-
 # Rebuild the builder image if necessary
   - name: 'gcr.io/cloud-builders/docker'
     args:
     - 'build'
     - '-t'
     - 'gcr.io/$PROJECT_ID/skaffold-builder:latest'
     - '-f'
-    - 'deploy/skaffold/Dockerfile'
-    - '--cache-from'
-    - 'gcr.io/$PROJECT_ID/skaffold-builder:latest'
-    - '--target'
-    - 'builder'
+    - 'deploy/skaffold/Dockerfile.int'
```

Comment:
it looks weird to use an integration Dockerfile in the release yaml. I think it's correct but maybe we can rethink the name?